types/node-temperature-converter/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "DOM"
```

Comment:
I need it for the tests. The only way you can test this is using `console.table`

Reply:
I don't understand how that could be the case.  The test would be fine without that line.  Regardless, if you want to include it, I believe you can use `/// <reference lib="dom"/>` in the test file to avoid pulling it into the product code.